enterprise_learner_portal/api/v1/views.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import get_object_or_404
+from django.utils.translation import ugettext as _
+
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from edx_rest_framework_extensions.auth.bearer.authentication import BearerAuthentication
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+
+from enterprise.models import EnterpriseCourseEnrollment, EnterpriseCustomerUser
+from enterprise.utils import NotConnectedToOpenEdX
+from enterprise_learner_portal.api.v1.serializers import EnterpriseCourseEnrollmentSerializer
+
+try:
+    from openedx.core.djangoapps.content.course_overviews.api import get_course_overviews
+except ImportError:
+    get_course_overviews = None
+
+User = get_user_model()
+
+
+class EnterpriseCourseEnrollmentView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    authentication_classes = (JwtAuthentication, BearerAuthentication, SessionAuthentication,)
+
+    def get(self, request):
+        if get_course_overviews is None:
+            raise NotConnectedToOpenEdX(
+                _('To use this endpoint, this package must be '
+                  'installed in an Open edX environment.')
+            )
+
+        user = request.user
+        enterprise_customer_user = get_object_or_404(
```

Comment:
i'm wondering if we should be mindful of having multiple enterprise's linked to the learner here. probably it would be best to follow up with separate work if we are trying to support this case... i will bring it up with jennifer to see her thoughts.

Reply:
we would like to have whatever we build support users linked to multiple enterprises, though we are not going to change any existing code unaffected by the scope of our project. i would recommend adding an enterprise id as a parameter passed into the path so that the endpoint will only return enrollments scoped to the enterprise. i leave it up to you to file a new story or complete the work with this story... i just don't want to forget about it  😄